CNI/romana

```diff
@@ -184,6 +187,52 @@ set_up_pod () {
 
 	# Kubernetes expects valid json on output so no one above should echo anything in stdout
 	echo -n $R
+
+	update_policy "$POD" "$NAMESPACE"
+}
+
+# update_policy discovers romana network policies that affect current pod
+# and notifies romana agent to include new pod into policy
+update_policy () {
```

Comment:
Yes, but: The policy definition does not include pods. It includes labels/segments, does it not? So the policy itself is not modified and exists already. We just realize here that a newly created pod matches those labels of the already existing policy.

So, the policy definition itself (some JSON similar to what was discussed in the network-SIG) does not change. Only what it applies to changes.

Therefore, 'update policy' implies something that doesn't actually happen.

I think we are just talking about the meaning of words here...?